src/file_pipe_log/log_file.rs

```diff
@@ -110,25 +190,42 @@ impl<F: FileSystem> LogFileWriter<F> {
     }
 }
 
+/// Build a file reader.
+///
+/// Attention please, the reader do not need a specified `[LogFileHeader]` from
+/// users.
+///
+/// * `[handle]`: standard handle of a log file.
+/// * `[expected_version]`: if `[None]`, reloads the log file header and parse
+/// the relevant `Version` before building the `reader`.
 pub(super) fn build_file_reader<F: FileSystem>(
     system: &F,
     handle: Arc<F::Handle>,
+    expected_version: Option<Version>,
```

Comment:
I could mistake `expected_version` for doing verification etc.
```suggestion
    version: Option<Version>,
```

Reply:
Done